src/components/PdfButton/PdfButton.cy.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import PdfButton, { formatFileName } from "@/components/PdfButton/PdfButton";
+import ShippingLabelsPdf, { ParcelClients } from "@/pdfs/ShippingLabels/ShippingLabelsPdf";
+
+const downloadsFolder = Cypress.config("downloadsFolder");
+const fileName = "ShippingLabels.pdf";
+const parcelClientsData: ParcelClients[] = [
+    {
+        packing_datetime: "19/06/2023",
+        collection_centre: "Delivery",
+        collection_datetime: "31/07/2023, 14:00",
+        voucher_number: "VOUCHER-NUMBER-2",
+        full_name: "Harry Potter",
+        phone_number: "07987654321",
+        address_1: "Hogwarts Castle",
+        address_2: "",
+        address_town: "Scottish Highlands",
+        address_county: "United Kingdom",
+        address_postcode: "YE11OW",
+        delivery_instructions: "Owls only no floo powder please.",
+    },
+    {
+        packing_datetime: "20/06/2023",
+        collection_centre: "Delivery",
+        collection_datetime: "31/08/2023, 14:00",
+        voucher_number: "VOUCHER-NUMBER-2",
+        full_name: "Nessie",
+        phone_number: "0799999999",
+        address_1: "Loch Ness",
+        address_2: "Inverness",
+        address_town: "Scottish Highlands",
+        address_county: "United Kingdom",
+        address_postcode: "NE55IE",
+        delivery_instructions: "Throw it in the water and leave as quickly as possible.",
+    },
+];
+
+const allFields = parcelClientsData
+    .map((obj) => {
+        return Object.values(obj);
+    })
+    .flat();
```

Comment:
will this not result in duplicate keys?

Reply:
No because: eg.

objs = [{"one": 1, "two": 2}, {"one": 3, "two": 4}]

the map will return
 [[1, 2], [3, 4]]

then the flat will do
[1,2,3,4]

No duplicate keys. Correct me if im wrong.